helper_uuids.go

```diff
@@ -0,0 +1,8 @@
+package recurly
+
+import "strings"
+
+// SanitizeUUID returns the uuid without dashes.
+func SanitizeUUID(id string) string {
+	return strings.TrimSpace(strings.Replace(id, "-", "", -1))
+}
```

Comment:
Can we move this into `subscriptions.go`? I know it's also used by transactions, but would like to avoid `helper`/`utils` files.